typescript/infra/scripts/warp-routes/transferRemote.sh

```diff
@@ -0,0 +1,234 @@
+#!/bin/zsh
+setopt KSH_ARRAYS
+xerc20s=(
+  "0x585afea249031Ea4168A379F664e91dFc5F77E7D"
+  "0x585afea249031Ea4168A379F664e91dFc5F77E7D"
+  "0x585afea249031Ea4168A379F664e91dFc5F77E7D"
+  "0x585afea249031Ea4168A379F664e91dFc5F77E7D"
+  "0x585afea249031Ea4168A379F664e91dFc5F77E7D"
+  "0x585afea249031Ea4168A379F664e91dFc5F77E7D"
+  "0x5EA461E19ba6C002b7024E4A2e9CeFe79a47d3bB"
+  "0x585afea249031Ea4168A379F664e91dFc5F77E7D"
+  "0x585afea249031Ea4168A379F664e91dFc5F77E7D"
+  "0x585afea249031Ea4168A379F664e91dFc5F77E7D"
+  "0x585afea249031Ea4168A379F664e91dFc5F77E7D"
+  "0x585afea249031Ea4168A379F664e91dFc5F77E7D"
+  "0x585afea249031Ea4168A379F664e91dFc5F77E7D"
+  "0x585afea249031Ea4168A379F664e91dFc5F77E7D"
+  "0x585afea249031Ea4168A379F664e91dFc5F77E7D"
+)
+
+warpRoutes=(
+  "0x3F536e156eD291c135ACb1D20F77C3B948E0F8a5"
+  "0xbA6BbAe1c1d25fFdd5bB74192d687A25d5e65326"
+  "0xBbcA34fF173339094cAc51e33BADeE86AA2C35b5"
+  "0x89E3530137aD51743536443a3EC838b502E72eb7"
+  "0x01bFbc80b32469c36DB4C7fc564E75475dfC278C"
+  "0xd4C1905BB1D26BC93DAC913e13CaCC278CdCC80D"
+  "0x9E3075E067932d744119e583B34d11b144CE1e4A"
+  "0xdfA407edf065b7ECfD95c3c5F4C32F9f34a5Fe80"
+  "0x21581dE0CB0Ce91E87b9d5124543C75Fa01ED9CC"
+  "0xa2656c131A9F204F73944D7B60DDB17565E71292"
+  "0x652e2F475Af7b1154817E09f5408f9011037492a"
+  "0xdD313D475f8A9d81CBE2eA953a357f52e10BA357"
+  "0x7712b534bF2b9fb7fA8D14EE83fDd077691a76C2"
+  "0x910FF91a92c9141b8352Ad3e50cF13ef9F3169A1"
+  "0xDF96074eefdD1Ae137C008bE165A3Ed4A55718FA"
+)
+
+safes=(
+  "0xA9421c6F339eC414b7e77449986bE9C2Ae430C25"
+  "0xA9421c6F339eC414b7e77449986bE9C2Ae430C25"
+  "0xA9421c6F339eC414b7e77449986bE9C2Ae430C25"
+  "0xA9421c6F339eC414b7e77449986bE9C2Ae430C25"
+  "0xA9421c6F339eC414b7e77449986bE9C2Ae430C25"
+  "0xf40b75fb85C3bEc70D75A1B45ef08FC48Db61115"
+  "0xA9421c6F339eC414b7e77449986bE9C2Ae430C25"
+  "0xA9421c6F339eC414b7e77449986bE9C2Ae430C25"
+  "0xf40b75fb85C3bEc70D75A1B45ef08FC48Db61115"
+  "0xf40b75fb85C3bEc70D75A1B45ef08FC48Db61115"
+  "0x31FF35F84ADB120DbE089D190F03Ac74731Ae83F"
+  "0xa30FF77d30Eb2d785f574344B4D11CAAe1949807"
+  "0xf40b75fb85C3bEc70D75A1B45ef08FC48Db61115"
+  "0xa7eccdb9be08178f896c26b7bbd8c3d4e844d9ba"
+  "0xf013c8Be28421b050cca5bD95cc57Af49568e8be"
+)
+
+# 'arbitrum',
+# 'optimism',
+# 'base',
+# 'blast',
+# 'bsc',
+# 'mode',
+# 'linea',
+# 'ethereum',
+# 'fraxtal',
+# 'zircuit',
+# 'taiko',
+# 'sei',
+# 'swell',
+# 'unichain',
+# 'berachain',
```

Comment:
still needed?

Reply:
Will convert this to a ts file in a bit. Mostly wanted this a placeholder

Reply:
ah you meant the comment. no dont need